packages/components/src/components/block/block.tsx

```diff
@@ -273,7 +273,7 @@ export class Block extends LitElement implements InteractiveComponent {
   /**
    * Fires when the component's header is clicked.
    *
-   * @deprecated Use `openClose` events such as `calciteBlockOpen`, `calciteBlockClose`, `calciteBlockBeforeOpen`, and `calciteBlockBeforeClose` instead.
+   * @deprecated in v3.0, removal target v6.0 - Use `openClose` events such as `calciteBlockOpen`, `calciteBlockClose`, `calciteBlockBeforeOpen`, and `calciteBlockBeforeClose` instead.
```

Comment:
Consistency suggestion:
```suggestion
   * @deprecated in v3.0.0, removal target v6.0.0 - Use `openClose` events such as `calciteBlockOpen`, `calciteBlockClose`, `calciteBlockBeforeOpen`, and `calciteBlockBeforeClose` instead.
```